src/app/app.module.ts

```diff
@@ -1,15 +1,15 @@
 import { BrowserModule } from "@angular/platform-browser";
 import { NgModule } from "@angular/core";
 
+import { ServicesModule } from "./core/index";
 import { AppRoutingModule } from "./app-routing.module";
-import { AppComponent } from "./app.component";
-
 import { DashboardLayoutModule } from "./dashboard-layout/index";
+import { AppComponent } from "./app.component";
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, DashboardLayoutModule, AppRoutingModule],
-  providers: [],
+  imports: [BrowserModule, DashboardLayoutModule, ServicesModule, AppRoutingModule],
+  providers: [ServicesModule],
```

Comment:
ServicesModule should not be in providers section.
Just import it.
But inside ServicesModule in providers section you have to list all the services